src/pretalx/orga/templates/orga/cfp/question_form.html

```diff
@@ -7,11 +7,17 @@
 
 {% block cfp_content %}
     {% compress js %}
-    <script defer src="{% static "js/jquery.formset.js" %}"></script>
-    <script defer src="{% static "cfp/js/animateFormset.js" %}"></script>
+        <script defer src="{% static "js/jquery.formset.js" %}"></script>
+        <script defer src="{% static "cfp/js/animateFormset.js" %}"></script>
+        <script defer src="{% static "vendored/moment-with-locales.js" %}"></script>
+        <script defer src="{% static "vendored/moment-timezone-with-data-2012-2022.js" %}"></script>
+        <script defer src="{% static "vendored/datetimepicker/bootstrap-datetimepicker.js" %}"></script>
+        <script defer src="{% static "orga/js/datetimepicker.js" %}"></script>
+        <script defer src="{% static "orga/js/questionRequiredField.js" %}"></script>
     {% endcompress %}
     <h2>
-        {% if form.instance.question %}{% translate "Question" %}: {{ form.instance.question }}{% else %}{% translate "New question" %}{% endif %}
+        {% if form.instance.question %}{% translate "Question" %}: {{ form.instance.question }}{% else %}
```

Comment:
Please revert this change.

Reply:
Here we have its same with the next comment.
We did not find a way to show the original author of the line with the git blame.

Reply:
As I said: Don't worry about it. I'll squash the changes into a single or a few commits before merging, and after that, git blame will work as expected, as if your change had never happened.